brands/amenity/fast_food.json

```diff
@@ -2903,6 +2903,21 @@
       "name": "Tropical Smoothie Cafe",
       "takeaway": "yes"
     }
+  },
+    "amenity/fast_food|Upper Crust": {
+     "countryCodes": ["de","dk","eg","es","fi","ie","gb","no","se"],
+      "matchTags": [ 
```

Comment:
Thanks!  You can actually remove the `matchTags` property here. 

We have a file [`match_groups.json`](https://github.com/osmlab/name-suggestion-index/blob/master/config/match_groups.json) which groups together similar tags, so that "amenity/fast_food" will automatically match other tags like "bakery" "fast_food" "cafe" "ice_cream" etc.

Reply:
I removed the matchtag in b27384a31